src/array_api_stubs/_draft/elementwise_functions.py

```diff
@@ -869,7 +869,7 @@ def conj(x: array, /) -> array:
     Parameters
     ----------
     x: array
-        input array. Should have a complex floating-point data type.
+        input array. Should have a floating-point data type.
```

Comment:
```suggestion
        input array. Should have a numeric data type.
```

We don't need to limit to only floating-point dtypes if providing a real-valued array is simply a no-op.

Reply:
A quick check.... yes, agreed:

```
In [47]: t = torch.arange(4, dtype=torch.uint8)

In [48]: torch.conj_physical(t).dtype
Out[48]: torch.uint8

In [49]: torch.real(t).dtype
Out[49]: torch.uint8

In [50]: a = np.arange(3, dtype=np.uint8)

In [51]: np.real(a).dtype
Out[51]: dtype('uint8')

In [52]: np.conj(a).dtype
Out[52]: dtype('uint8')

```